config.py-bak

```diff
@@ -0,0 +1,58 @@
+# ######### Django settings ###########
```

Comment:
do we need this file in the repo?

Reply:
ok I deleted, i think we're all set for the is pull request to go through